src/genericBlogPost.tsx

```diff
@@ -29,7 +29,7 @@ import { Store } from 'redux'
 
 export async function Mount({ store }: { store: Store<State, Actions> }) {
   const state = store.getState()
-  const pages = getPages(state).filter(it => !hasTag(state, 'bite-sized')(it))
+  const pages = getPages(state).filter(hasTag(state, 'general-post'))
   await Promise.all(
```

Comment:
Should add a tag to the general post, else `getPages` will get all pages, included the non-blog-post pages.